### ISSUE TITLE
Individual asset view

### ISSUE THREAD

**Issue by jenbennings** [lionsharecapital/lionshare-desktop#54]
<img width="771" alt="screen shot 2017-02-18 at 5 40 30 pm" src="https://cloud.githubusercontent.com/assets/3468612/23098412/6506120c-f601-11e6-9b20-b9a2bf13ea7c.png">

Expanded view for each asset which provides additional info (provided by the API). Accessible by clicking the asset in the prices list.

Still an early concept, open to thoughts/suggestions!

cc: @jorilallo @maksim-s @jessepollak

**Comment by jenbennings** [comment]
<img width="1397" alt="screen shot 2017-02-19 at 2 56 55 pm" src="https://cloud.githubusercontent.com/assets/3468612/23107567/1c63754a-f6b4-11e6-816d-c8d572beac3a.png">

**Comment by jessepollak** [comment]
I really like this — just to clarify, do you see the description hiding when a user has a position vs. showing when they don't?

**Comment by jorilallo** [comment]
Big fan of this. I would really like to see all this being available in the API, would help out other apps as well.

**Comment by jenbennings** [comment]
@jessepollak Yep, I imagine if people had it in their portfolio they wouldn't need it.

**Comment by mickaphd** [comment]
Big fan also!
